src/AmbientSounds.Uwp/MultilingualResources/AmbientSounds.Uwp.pt-BR.xlf

```diff
@@ -0,0 +1,256 @@
+        </trans-unit>
+        <trans-unit id="SuggestionHeader.Text" translate="yes" xml:space="preserve">
+          <source>Want more sounds? Send suggestions!</source>
+          <target state="translated">Quer mais sons? Envie sugestões!</target>
+        </trans-unit>
+        <trans-unit id="SuggestionTextBox.PlaceholderText" translate="yes" xml:space="preserve">
+          <source>Suggestion</source>
+          <target state="translated">Sugestões</target>
+        </trans-unit>
+        <trans-unit id="SuggestionTextBox.AutomationProperties.Name" translate="yes" xml:space="preserve">
+          <source>Want more sounds? Send suggestions!</source>
+          <target state="translated">Quer mais sons? Envie sugestões!</target>
+        </trans-unit>
+        <trans-unit id="SuggestionSendButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
+          <source>Send suggestion</source>
+          <target state="translated">Enviar sugestão</target>
+        </trans-unit>
+        <trans-unit id="SuggestionSendButton.AutomationProperties.Name" translate="yes" xml:space="preserve">
+          <source>Click to send the suggestion</source>
+          <target state="translated">Clique para enviar a sugestão</target>
+        </trans-unit>
+        <trans-unit id="ThankYouTextBlock.Text" translate="yes" xml:space="preserve">
+          <source>Thank you!</source>
+          <target state="translated">Obrigado!</target>
+        </trans-unit>
+        <trans-unit id="SettingsThemeHeader.Text" translate="yes" xml:space="preserve">
+          <source>Theme (requires restart)</source>
+          <target state="translated">Tema (requer reinicialização)</target>
+        </trans-unit>
+        <trans-unit id="SettingsThemeDefaultRadio.Content" translate="yes" xml:space="preserve">
+          <source>Auto</source>
+          <target state="translated">Automático</target>
+        </trans-unit>
+        <trans-unit id="SettingsThemeDarkRadio.Content" translate="yes" xml:space="preserve">
+          <source>Dark</source>
+          <target state="translated">Escuro</target>
+        </trans-unit>
+        <trans-unit id="SettingsThemeLightRadio.Content" translate="yes" xml:space="preserve">
+          <source>Light</source>
+          <target state="translated">Claro</target>
+        </trans-unit>
+        <trans-unit id="Sound-vacuum" translate="yes" xml:space="preserve">
+          <source>Vacuum</source>
+          <target state="translated">Vácuo</target>
+        </trans-unit>
+        <trans-unit id="Sound-thunder" translate="yes" xml:space="preserve">
+          <source>Thunder</source>
+          <target state="translated">Trovões</target>
+        </trans-unit>
+        <trans-unit id="SettingsNotificationsSwitch.Header" translate="yes" xml:space="preserve">
+          <source>Notification for new sounds</source>
+          <target state="translated">Notificação para novos sons</target>
```

Comment:
There are a couple new English strings in the latest main branch. I think they're for the new Fan and Wind sounds. Can you merge from main and translate those strings?